src/main/java/net/discordjug/javabot/listener/filter/MessageContent.java

```diff
@@ -0,0 +1,20 @@
+package net.discordjug.javabot.listener.filter;
+
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+
+import java.util.List;
+
+/**
+ * Message content.
```

Comment:
Javadocs should describe what the documented elements do. Please describe both what the record is representing and what the components are.